src/pkg/util/capabilities/capabilities.go

```diff
@@ -408,5 +408,22 @@ func Split(caps string) ([]string, []string) {
 		included = append(included, c)
 	}
 
-	return included, excluded
+	return RemoveDuplicated(included), RemoveDuplicated(excluded)
+}
+
+// RemoveDuplicated removes duplicated capability value from
+// provided list and returns it
+func RemoveDuplicated(caps []string) []string {
```

Comment:
I think a simplification could have been to use a map of string to bool and populate your string list from the map results.